branchConnectivityConfiguration addition (#22277)
microsoftgraph/microsoft-graph-docs: api-reference/beta/resources/networkaccess-connectivityconfigurationlink.md

---
title: "connectivityConfigurationLink resource type"

> description: "Represents the link or association between a branch, and its corresponding connectivity configuration."

Reviewer: Is link referring to "deviceLink"?

author: "Moti-ba"
ms.localizationpriority: medium
ms.prod: identity-and-access
doc_type: resourcePageType
---

# connectivityConfigurationLink resource type

Namespace: microsoft.graph.networkaccess

[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]

> Represents the link or association between a branch, and its corresponding connectivity configuration.

Reviewer: Is this technically accurate?
```suggestion
Represents the connectivity configuration of a [deviceLink](../resources/networkaccess-devicelink.md) in a branch.
```

## Methods

> No methods available.

Reviewer: ```suggestion
None.
```

## Properties
|Property|Type|Description|
|:---|:---|:---|
|displayName|String|Specifies the name of the link.|
|id|String|A unique identifier for each link.|
|localConfigurations|[microsoft.graph.networkaccess.localConnectivityConfiguration](../resources/networkaccess-localconnectivityconfiguration.md) collection|Specifies configuration of Microsoft’s end of the tunnel(s) for a device link.|

> |peerConfiguration|[microsoft.graph.networkaccess.peerConnectivityConfiguration](../resources/networkaccess-peerconnectivityconfiguration.md)|Specifies configuration of customer’s end of the tunnel(s) for a device link.|

Reviewer: - If link means "deviceLink", change the references to deviceLink.
- What is end of the tunnels? On performing a quick search, I can't find any reference to this term? Will devs understand it? Is there a more standard term you can use instead?
